src/Services/Air/AirParser.js

```diff
@@ -531,6 +520,55 @@ const airGetTicket = function (obj) {
   return getTicketFromEtr.call(this, etr, obj);
 };
 
+function airGetTicketsErrorHandler(rsp) {
+  let errorInfo;
+  let code;
+  try {
+    errorInfo = (
+      rsp.detail[`common_${this.uapi_version}:ErrorInfo`]
+      || rsp.detail['air:AvailabilityErrorInfo']
+    );
+    code = errorInfo[`common_${this.uapi_version}:Code`];
+  } catch (err) {
+    throw new RequestRuntimeError.UnhandledError(null, new AirRuntimeError(rsp));
+  }
+  // General Air Service error
+  if (code === '3000') {
+    if (
+      rsp.faultcode !== undefined
+      && rsp.faultstring !== undefined
+      && /has no tickets/.test(rsp.faultstring) === true
+    ) {
+      return rsp;
+    }
+  }
+  switch (code) {
+    case '345':
```

Comment:
If it's the only one case then maybe `if/else` will be better?

Reply:
Unless it's possible to have more cases in the future.

Reply:
More cases could be added.